yt_dlp/extractor/ard.py

```diff
@@ -630,13 +632,36 @@ def _real_extract(self, url):
     }
   }
 }''' % (client, video_id),
-            }).encode(), headers={
-                'Content-Type': 'application/json'
-            })['data']['playerPage']
+                }).encode(), headers={
+                    'Content-Type': 'application/json'
+                })['data']['playerPage']
+
+            content_id = str_or_none(try_get(
+                player_page, lambda x: x['tracking']['atiCustomVars']['contentId']))
```

Comment:
This code is duplicated between the two blocks. We should move it out of the `try:` `except:`. Imo the easiest way would be to set both player_page as well as raw_player_page or sth